src/services/edusharing/hooks/index.js

```diff
@@ -0,0 +1,27 @@
+const auth = require('@feathersjs/authentication');
+const { disallow } = require('feathers-hooks-common');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'feathers-hooks-common'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4791331)